js/binance.js

```diff
@@ -1201,13 +1230,12 @@ module.exports = class binance extends Exchange {
     }
 
     async fetchTime (params = {}) {
-        const defaultType = this.safeString2 (this.options, 'fetchTime', 'defaultType', 'spot');
-        const type = this.safeString (params, 'type', defaultType);
+        const type = this.getDefaultType ('fetchTime', 'params');
```

Comment:
as pointed by @ttodua we're passing here params as a literal string instead of the variable